src/stan/agrad/autodiff.hpp

```diff
@@ -359,6 +357,75 @@ namespace stan {
       stan::agrad::recover_memory_nested();
     }
 
-  }
-}
+    /**
+     * Calculate the value, the Hessian, and the gradient of the Hessian
+     * of the specified function at the specified argument.
+     *
+     * <p>The functor must implement
+     *
+     * <code>
+     * using stan::agrad::fvar;
+     * using stan::agrad::var;
+     *
+     * fvar<fvar<var> >
+     * operator()(const
+     * Eigen::Matrix<fvar<fvar<var> >, Eigen::Dynamic, 1>&)
+     * </code>
+     *
+     * using only operations that are defined for
+     * <code>stan::agrad::fvar</code> and <code>stan::agrad::var</code>.
+     *
+     * This latter constraint usually
+     * requires the functions to be defined in terms of the libraries
+     * defined in Stan or in terms of functions with appropriately
+     * general namespace imports that eventually depend on functions
+     * defined in Stan.
+     *
+     * @tparam F Type of function
+     * @param[in] f Function
+     * @param[in] x Argument to function
+     * @param[out] fx Function applied to argument
+     * @param[out] H Hessian of function at argument
```

Comment:
When I run make doxygen, I get the following error:

RobTrangucci ~/stan 18:56 [
feature/issue-1272-gradient-of-hessian-autodiff_hpp*] - 593:

$ make doxygen

mkdir -p doc/api

doxygen src/doxygen/doxygen.cfg

make: doxygen: No such file or directory

make: *** [doxygen] Error 1


Rob

On Wed, Feb 11, 2015 at 8:19 AM, Daniel Lee <notifications@github.com>
wrote:

> In src/stan/agrad/autodiff.hpp
> <https://github.com/stan-dev/stan/pull/1276#discussion_r24493619>:
>
> > +     * </code>
> > +     *
> > +     * using only operations that are defined for
> > +     * <code>stan::agrad::fvar</code> and <code>stan::agrad::var</code>.
> > +     *
> > +     * This latter constraint usually
> > +     * requires the functions to be defined in terms of the libraries
> > +     * defined in Stan or in terms of functions with appropriately
> > +     * general namespace imports that eventually depend on functions
> > +     * defined in Stan.
> > +     *
> > +     * @tparam F Type of function
> > +     * @param[in] f Function
> > +     * @param[in] x Argument to function
> > +     * @param[out] fx Function applied to argument
> > +     * @param[out] H Hessian of function at argument
>
> Doxygen is balking at the comments. Do you have oxygen installed? Type make
> doxygen
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/stan-dev/stan/pull/1276/files#r24493619>.
>